### ISSUE TITLE
compile error

### ISSUE THREAD

**Issue by minershaven** [49handyman/EquihashNompPool#10]
while trying to build the pool it exits out with the error

`751 error Error while executing:
751 error /usr/bin/git ls-remote -h -t https://github.com/BitGo/blake2b-wasm.git
751 error
751 error remote: Support for password authentication was removed on August 13, 2021.`

**Comment by minershaven** [comment]
isolated the error to the stratum-pool module in the package.json but cant seem to find what it is in there that's causing the problem

**Comment by mkphaber** [comment]
Yes, I encountered the same problem. We cannot run the pool because we cannot load the module.
